substrate-airgapped/Cargo.toml

```
@@ -9,15 +9,18 @@ edition = "2018"
 codec = { package = "parity-scale-codec", version = "1.3.5", default-features = false }
 
 # Substrate (hopefully these can be removed in the future)
-sp-runtime = { version = "2.0.0", default-features = false }
-sp-core = { version = "2.0.0", default-features = false }
+sp-runtime = { version = "2.0.1", default-features = false }
+sp-core = { version = "2.0.1", default-features = false }
+
+primitive-types = { version = "0.7", default-features = false, features = ["rustc-hex", "serde"] }
+anyhow = "1"
 
 [dev-dependencies]
 # Currently these are just to get the example going
-sp-keyring = { version = "2.0.0", package = "sp-keyring", default-features = false }
-sp-version = { version = "2.0.0", package = "sp-version" }
+sp-keyring = { version = "2.0.1", package = "sp-keyring", default-features = false }
```

redundant package-name

```suggestion
sp-keyring = { version = "2.0.1", default-features = false }
```